.woodpecker/stg.yml

```diff
@@ -31,9 +33,9 @@ steps:
         - git pull
         - itkdev-docker-compose-server up -d --force-recreate --remove-orphans
         # Build theme
-        - itkdev-docker-compose-server run --rm node yarn --cwd /app/web/themes/custom/itkdev/itkdev_project_theme install
-        - itkdev-docker-compose-server run --rm node yarn --cwd /app/web/themes/custom/itkdev/itkdev_base_theme install
-        - itkdev-docker-compose-server run --rm node yarn --cwd /app/web/themes/custom/itkdev/itkdev_base_theme build
+        - itkdev-docker-compose-server run --rm -e PLUGIN_FONTAWESOME_PACKAGE_TOKEN node yarn --cwd /app/web/themes/custom/itkdev/itkdev_project_theme install
```

Comment:
```suggestion
        - itkdev-docker-compose-server run --rm -e FONTAWESOME_PACKAGE_TOKEN=${PLUGIN_FONTAWESOME_PACKAGE_TOKEN} node yarn --cwd /app/web/themes/custom/itkdev/itkdev_project_theme install
```

Reply:
Remembered that we need to more here because node expects the env to be named `FONTAWESOME_PACKAGE_TOKEN` but Woodpecker will prefix with `PLUGIN_`

See https://woodpecker-ci.org/docs/usage/secrets#usage